add docs for multi-modal examples (#1227)
huggingface/diffusers: docs/source/using-diffusers/other-modalities.mdx

<!--Copyright 2022 The HuggingFace Team. All rights reserved.

Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
the License. You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
specific language governing permissions and limitations under the License.
-->

# Using Diffusers with other modalities

Diffusers is in the process of expanding to modalities other than images.

> Currently, the only example is for [molecule conformation](https://www.nature.com/subjects/molecular-conformation#:~:text=Definition,to%20changes%20in%20their%20environment.) generation.

Reviewer: ```suggestion
Currently, one example is for [molecule conformation](https://www.nature.com/subjects/molecular-conformation#:~:text=Definition,to%20changes%20in%20their%20environment.) generation.
```

We also have audio, don't we? Maybe cross-link there too?

Author: I made a entire page for audio as I think it'll grow (much like I made a page for RL)
